.drone.yml

```
@@ -0,0 +1,30 @@
+kind: pipeline
+name: arm64
+platform: { os: linux, arch: arm64 }
+steps:
+- name: Test
+  image: buildpack-deps:focal
+  commands:
+    - export LC_ALL=C.UTF-8
+    - apt-get update -y
+    - apt-get install -y ghc cabal-install
+    - ghc --version
+    - cabal --version
+    - cabal new-update
+    - cabal new-test
```

I think that should take care of doctest failure, right?
```suggestion
    - cabal new-test --write-ghc-environment-files=always
```